plugins/badges/src/api/BadgesClient.ts

```diff
@@ -53,7 +52,7 @@ export class BadgesClient implements BadgesApi {
 
   private async getEntityBadgeSpecsUrl(entity: Entity): Promise<string> {
     const routeParams = this.getEntityRouteParams(entity);
-    const path = generatePath(entityRoute.path, routeParams);
+    const path = generatePath(`:kind/:namespace/:name`, routeParams);
```

Comment:
Really need to use `${encodeURIComponent(kind)}/...` etc. here tbh, since a `/` in the name would break things atm

Reply:
Ah wait not it wouldn't, we're templating in a different way here :grin:

So fine as is, wonder if it's something we should encourage in fact, it's bit of a shame to standardize on something here that doesn't work in the backend